### ISSUE TITLE
Hardcoded URL in Grafana, slightly broken layout

### ISSUE THREAD

**Issue by jointerCZ** [tcivie/meshtastic-metrics-exporter#66]
Hello,
you have hardcoded URL in the Main dashboard, when I click on the General Information section ID node, the dashboard opens from .it page.
Also the Main layout seems to be a bit incomplete, some values match the screenshot and some are missing.
I would prefer to have the maximum possible amount of data there, I can always delete it, but finding useful data takes more time...
But great work.
Regards

**Comment by tcivie** [comment]
Hey @jointerCZ - thanks for bringing this up! I've modified all dashboard links to target `localhost:3000` instead and refreshed the dashboards to display more comprehensive information.

The provided dashboards are meant as a starting point - feel free to customize them to your needs. The `README` has been updated with a complete list of all metrics available through the exporter, which should help you create even more complex and useful boards.

If you create any interesting dashboard configurations, we'd love to see them! Feel free to fork the project and open a PR with your changes. The community would definitely benefit from more dashboard examples.
Let me know if you need any help or have more suggestions!

**Comment by jointerCZ** [comment]
Hello,
thanks for quick fix.
I found somewhere you can use relative links like /d/edqo1uh0eglq8g/node-dashboard?orgId=1&var-nodeID=${__data.fields[0]} so all you need to do is to delete the front part with the domain and port. That is how I fixed it initially.

I do not have much experience with git, could you describe what commands I need to run to clone it again and "overwrite" the templates in the Grafana container?

I am using Grafana for SNMP and Home Assistant graphs, but not advanced enough to make something awesome from this data.
You can however search for similar stats in Grafana for msh/CZ for inspiration. Not sure who done them but it is similar to yours but still different, so probably not just an instance of this.

**Comment by tcivie** [comment]
Hey @jointerCZ to contribute I would suggest looking at this Github Guide on [how to contribute to projects](https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project) 

After you have cloned the project and ran the grafana locally, you can make the changes you want to the dashboards, then you can copy their JSON data (Directcly from the Grafana UI) and update it in your cloned project under the `docker/grafana/provisioning/dashboards` directory.

Then you can submit a PR with the details on what you changed/added (With some images if you want)

Regarding your proposal for removing the url' I'll defintely look into it.

**Comment by jointerCZ** [comment]
Thanks, I deleted it, cloned it again, ran
docker rm -f 34b955cdf5a0
docker volume rm meshtastic-metrics-exporter_grafana_data
docker compose up

So I kept the data but have fresh Grafana and dashboards, that is what I wanted.
See this, do it in next release as I had to fix the localhost again:
https://stackoverflow.com/questions/61612381/how-do-you-make-links-in-grafana-table-panel-from-a-cell-relative-so-they-can-s

I would like to have device firmware version in the Investigation /  General Information, I will need to take a look if I can get that somehow.
Also the Max hop section is a mess for me with 500+ nodes from MQTT in one day.
See https://abacus.sh.cvut.cz/grafana/dashboards for possible inspiration. (not sure where it comes from)

Again, wonderful packaging and simple to install everything, first time in years I do not have to spend hours to troubleshoot multiple packages and interconnections.
Now I want this, to have info about my locally connected node only and that will take much more time.
https://github.com/cordelster/mesh-metrics
Best regards
